media/css/firefox/family-nav.less

```diff
@@ -4,6 +4,12 @@
 
 @import "../sandstone/lib.less";
 
+#tabzilla {
+    &:before {
+        background-color: @textColorSecondary;
+    }
+}
+
```

Comment:
This selector isn't necessary as the file stands. The family nav has its own tabzilla implementation (down on line 57). However, it probably is best to switch the family nav over to the standard implementation...which would require a little bit more work. Could be broken out to a separate PR - up to you.